react/src/components/VFolderNodes.tsx

```diff
@@ -215,6 +337,58 @@ const VFolderNodes: React.FC<VFolderNodesProps> = ({
         ]}
         {...tableProps}
       />
+      <BAIConfirmModalWithInput
+        open={!!currentVFolder}
+        onOk={() => {
+          deleteFromTrashBinMutation.mutate(currentVFolder?.id ?? '', {
+            onSuccess: () => {
+              onRequestChange?.();
+              message.success(
+                t('data.folders.FolderDeletedForever', {
+                  folderName: currentVFolder?.name,
+                }),
+              );
+            },
+            onError: (error) => {
+              upsertNotification({
+                description: painKiller.relieve(error?.message),
+                open: true,
+              });
+            },
+          });
+          setCurrentVFolder(null);
+        }}
+        onCancel={() => {
+          setCurrentVFolder(null);
+        }}
+        confirmText={currentVFolder?.name ?? ''}
+        content={
+          <Flex
+            direction="column"
+            gap="md"
+            align="stretch"
+            style={{ marginBottom: token.marginXS, width: '100%' }}
+          >
+            <Alert
+              type="warning"
+              message={t('dialog.warning.DeleteForeverDesc')}
+              style={{ width: '100%' }}
+            />
+            <Flex>
+              <Typography.Text style={{ marginRight: token.marginXXS }}>
+                {t('data.folders.TypeFolderNameToDelete')}
+              </Typography.Text>
+              (
+              <Typography.Text code copyable>
```

Comment:
How about removing `copyable` ? Because this is a very serious destroying action, it's better not to have this button.